modules/op-release-notes-1-9.adoc

```diff
@@ -77,7 +77,14 @@ In addition to the fixes and stability improvements, the following sections high
 [id="operator-new-features-1-9_{context}"]
 === Operator
 
-* With this update, you can install {pac} as a separate component, not as a part of `TektonAddon`. You can configure {pac} in the `TektonConfig` CRD.
+* With this update, {pac} is installed by default and it is not a part of `TektonAddon`. You can disable {pac} by using the following flag:
```

Comment:
I am thinking we should avoid mentioning about `TektonAddon`  as it may confuse to new PAC user 

@vdemeester wdyt for below sentence 
```suggestion
* With this update, {pac} is installed by default. You can disable {pac} by using the following flag:
```

Reply:
@dhruv, please see the guidance wrt Kubernetes Objects here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references especially the second note and the examples above it. That is relevant if you retain the words - `TektonAddon` If not please read and apply when you use CRDs.

Reply:
Wrong dev

Reply:
@Preeticp - I have decided to remove the words TektonAddon as instructed by the SME for the time being.